packages/cubejs-databricks-jdbc-driver/src/DatabricksDriver.ts

```diff
@@ -142,6 +142,30 @@ export class DatabricksDriver extends JDBCDriver {
     return !!this.config.readOnly;
   }
 
+  public showDeprecationWarnings() {
+    this.showUrlTokenDeprecation();
+  }
+
+  public async testConnection(): Promise<void> {
```

Comment:
Could you please move this warning to a place where it will be shown once and not on every connection?